### PR TITLE
Update README and add USAGE

### PR DESCRIPTION
**Pull request by npendery**

### CODE REVIEW

File: docs/USAGE.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+
+### Pushing changes
+
+You can use `git` as you normally would, either on your own host machine or in Docker's `web` container.
+
+
+### Serving Ember
+
+The Code Corps API is intended to work alongside a client written in Ember. For that purpose, the Rails application exposes all of its API endpoints behind an `api.` subdomain.
```
Comment (rigelstpierre) [inline docs/USAGE.md]: Maybe mention something about the API being rewritten in Phoenix.

**Comment by rigelstpierre** [comment]
Except for my comments looks good 🐑 🇮🇹

File: README.md [hunk 2]
```diff
@@ -4,6 +4,13 @@
 
 [![Build Status](https://semaphoreci.com/api/v1/joshsmith/code-corps-phoenix/branches/add-users-endpoint/badge.svg)](https://semaphoreci.com/joshsmith/code-corps-phoenix)
 
+
+## Installing with Docker
+
+To make your life easier, you can just clone this repository and use our Docker container. [Follow this guide to get started.](docs/INSTALLING.md)
+
+## Running the app without Docker
```
Comment (joshsmith) [inline README.md]: We might want to remove this section altogether. I would rather not encourage people to run the app without Docker. It's the happy path.

File: docs/USAGE.md (hunk 1 above)
Comment (joshsmith) [inline docs/USAGE.md]: Yes, change `Rails` to `Phoenix` here.

File: docs/USAGE.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+On the Ember client side of things, we use [`ember-cli-deploy`](https://github.com/ember-cli/ember-cli-deploy) with a `redis` plugin to deploy the client application to redis. Multiple revisions are maintained this way.
+
+Any server request pointing to the main domain and not the `api.` subdomain is redirected to the API's `ember_index_controller#index`. There, depending on the remainder of the request path and the current environment, a specific revision of the Ember app's `index.html` is retrieved from redis and rendered. This `index.html` can be:
+* the development revision, if the current environment is development
+* a specific deployed revision in production if the request contains a revision parameter in SHORT_UUID format
+* the latest deployed revision in production if the request does not contain a revision parameter
+* a plain text string containing "INDEX NOT FOUND" if a revision was specified, but the key for the specified revision was not found by redis
```
Comment (joshsmith) [inline docs/USAGE.md]: Actually, I would kill all mention of "Serving Ember" for now. None of it is relevant yet.

**Comment by joshsmith** [comment]
@npendery with those changes, will be good to go.

**Pull request merged**